### ISSUE TITLE
Support YMF - YAML Front Matter

### ISSUE THREAD

**Issue by maricn** [fannheyward/coc-markdownlint#215]
It would be nice if this extension didn't complain with `MD025/single-title/single-h1` when using YAML front matter block ([as Jekyll does](https://jekyllrb.com/docs/front-matter/), but as any custom YMF can be defined).

Example:
```markdown
---
tags: ['foo-tag']
title: Test article
created: '2019-12-16T13:02:06.643Z'
modified: '2020-04-07T12:25:37+0200'
---

# Test title h1
```

**Comment by fannheyward** [comment]
Add `"MD025": { "front_matter_title": "" }` to your markdownlint config. FYI https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md025
